app/views/recipes/show.html.erb

```diff
@@ -0,0 +1,15 @@
+<%= render 'shared/navbar' %>
+<div class="container">
+  <p style="color: green"><%= notice %></p>
+  <h1 class="title"><%= @recipe.name %></h1>
+
+    <%= render 'recipe-show' %>
+
+  <div class='recipe-buttons' >
+      <%= button_to "Generate Shopping List" %>
```

Comment:
- [ ] When clicking on `Generate Shopping List` it shows redirects the user to the shopping list page that matches the given [wireframe](https://raw.githubusercontent.com/microverseinc/curriculum-rails/main/recipe-app/images/general_shopping_list.png?token=GHSAT0AAAAAABVWKLFQH2WS5DUATR5ZR3ZYY3NRBJQ). Currently the button is not working and it would be good if you can take a look at it.

Reply:
Thanks